torch_np/_wrapper.py

```diff
@@ -824,6 +824,16 @@ def take_along_axis(arr: ArrayLike, indices: ArrayLike, axis):
     return result
 
 
+@normalizer
+def take(a: ArrayLike, indices: ArrayLike, axis=None, out=None, mode="raise"):
+    if out is not None:
+        raise NotImplementedError(f"{out=}")
```

Comment:
Also OK for now. Will likely get resolved automagically by gh-93, which is still in flux